TestResultSummaryService/parsers/BenchmarkMetric.js

```diff
@@ -48,6 +48,15 @@ const BenchmarkMetricRegex = {
                 regexRepeat: false
             }
         ]
+    },
+    AcmeAir_Nodejs: {
+        metrics: [
+            {
+                name: "Global Throughput",
+                regex: /(\d+.\d+)\/s/,
```

Comment:
Might be good to add some unique string in your benchmark output before the metric value for easier identification as done for other benchmark metrics. Also, can you please post the actual string that you're trying to parse from the output?

Reply:
Regex like `regex: /Throughput....` would be more meaningful. Would be good to add a comment like `Example: "Throughput: 3554.1"` next to the regex.

Reply:
Thank you for the suggestion. Changed "Global Throughput" to "Throughput" and commented the output that needs to be extracted next to regex.